src/examples/chat/client.cc

```diff
@@ -30,7 +30,7 @@ int main(int argc, char* argv[]) {
   zmq::context_t context(1);
 
   std::vector<std::tuple<server_address_t, client_address_t, nickname_t>>
-      connect_tuple = {{server_address, client_address, nickname}};
+      connect_tuple = std::vector<std::tuple<server_address_t, client_address_t, nickname_t>>{std::tuple<server_address_t, client_address_t, nickname_t>{server_address, client_address, nickname}};
```

Comment:
Hm, I'm having trouble reproducing this on wandbox. The following code:

```cpp
#include <tuple>
#include <vector>
#include <string>

int main() {
    std::string x;
    std::string y;
    std::string z;
    std::vector<std::tuple<std::string, std::string, std::string>> xs = {{x, y, z}};
}
```

builds on all the versions of clang++ and g++ that I tried. The [tuple constructors](http://en.cppreference.com/w/cpp/utility/tuple/tuple) do look explicit to me, but I admit I don't understand explicit constructors and initializer lists and explicit and all that well enough to understand what's going on.